audio-matching/utils/dbHelper.py

```diff
@@ -0,0 +1,43 @@
+import mysql.connector
+import secrets
+from mysql.connector import errorcode
+
+config = {
+  'user': secrets.user,
+  'password': secrets.password,
+  'host': secrets.host,
+  'database': secrets.db,
+  'raise_on_warnings': True
+}
+
+def urlExists(url):
+  cnx = mysql.connector.connect(**config)
+  cursor = cnx.cursor()
+
```

Comment:
`labelled` is a sensical name. But over the past few days I have been thinking on and off on a name for our `big mama` supervised table ... Any other candidates ? @tjlam @Eldrick19